### PR TITLE
CASMCMS-7693: Announce CRUS deprecation

### PR DESCRIPTION
**Pull request by jsollom-hpe**
## Summary and Scope

Announce CRUS deprecation## Issues and Related PRs

_List and characterize relationship to Jira/Github issues and other pull requests. Be sure to list dependencies._

* Resolves CASMCMS-7693

## Testing
Nope, it's documentation.

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

**Review by ecozzi-hpe — APPROVED** [review]

**Review by rbak-hpe — APPROVED** [review]

**Review by jsl-hpe — APPROVED** [review]

File: introduction/differences.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ The most noteworthy changes since the previous release are described here.
         * The cray-dns-unbound-manager CronJob will be deprecated in a future release once all DNS records are migrated to PowerDNS.
         * The introduction of PowerDNS and Bifurcated CAN will introduce some node and service naming changes.
         * Please see the [PowerDNS migration notice](../introduction/PowerDNS_migration.md) for more information.
+   * The Compute Rolling Upgrade Service (CRUS) will be deprecated in the CSM-1.3.0 release. Enhanced BOS functionality will replace CRUS. This includes the ability to stage changes to nodes that can be acted upon later when the node reboots. It also includes the ability to reboot nodes without specifying any boot artifacts. This latter ability relies on the artifacts already having been staged.
```
Comment (rustydb) [inline introduction/differences.md:51]: This doesn't seem very emphasized for something that seems like an important note for admins.
Perhaps prefixing it with 
```markdown
**`NOTE`**
```
Or bolding the first sentence would do it more justice?

Comment (jsollom-hpe) [reply introduction/differences.md:51]: > 

@rustydb , are you talking about my addition needing more emphasis or the entire section as a whole? I just followed the established pattern when I added mine.

Comment (rustydb) [reply introduction/differences.md:51]: The addition.

Comment (rustydb) [reply introduction/differences.md:51]: _looks again at the other contents in the file_

I suppose there are other deprecations in here already, so maybe this is totally moot.

**Review by phalseth-hpe — APPROVED** [review]

**Pull request merged**